Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
Why 'T'? Why not 'E'?
We deal her wit Extruders not Temperatures.

Couldn't you generate E1X and only add "Offset" as a string?